test/unittests/eof_validation_test.cpp

```diff
@@ -677,55 +673,52 @@ TEST_F(eof_validation, deprecated_instructions)
 
 TEST_F(eof_validation, max_arguments_count)
 {
-    add_test_case("EF0001 010008 02000200830001 040000 00 0080007F 7F7F007F" +
-                      127 * bytecode{"5F"} + "E3000100 E4",
+    add_test_case(
+        eof_bytecode(127 * push0() + callf(1) + OP_STOP, 127).code(OP_RETF, 127, 127, 127),
         EOFValidationError::success);
 
-    add_test_case("EF0001 010008 02000200010001 040000 00 00800000 80800080 00 E4",
+    add_test_case(eof_bytecode(OP_STOP, 0).code(OP_RETF, 128, 128, 128),
         EOFValidationError::inputs_outputs_num_above_limit);
 
-    add_test_case("EF0001 010008 020002000400FF 040000 00 0080007F 007F007F E3000100" +
-                      127 * bytecode{1} + OP_RETF,
+    add_test_case(
+        eof_bytecode(callf(1) + OP_STOP, 127).code(127 * bytecode{1} + OP_RETF, 0, 127, 127),
         EOFValidationError::success);
 
-
-    add_test_case(bytecode{"EF0001 010008 02000200010101 040000 00 00800000 00810081"} + OP_STOP +
-                      128 * bytecode{1} + OP_RETF,
+    add_test_case(eof_bytecode(OP_STOP).code(128 * bytecode{1} + OP_RETF, 0, 129, 129),
```

Comment:
I think the original intention was to push 129 values, probably this got broken during non-returning functions introduction. Also we should make the 2nd code section reachable.
```suggestion
    add_test_case(eof_bytecode(callf(1) + OP_STOP).code(129 * push(1) + OP_RETF, 0, 129, 129),
```